[Marktplaats] New Bridge

Park0 opened the pull request.

Park0 committed:
Support Marktplaats.nl search

Park0 committed:
Formatting: Removed space

em92 commented:
Please name your PR properly. https://github.com/RSS-Bridge/rss-bridge/wiki/Pull-request-policy. Examples: https://github.com/RSS-Bridge/rss-bridge/commits/master

And lint the code: https://travis-ci.org/github/RSS-Bridge/rss-bridge/jobs/691256961#L535-L542

Park0 commented:
Done

Park0 committed:
More parameters added

Added option to filter on distance & price
Added option to filter global items without location

teromene commented:
Hello and welcome to the project !
I am fine with the code, are you ready for merge ?

Park0 commented:
Yes i am, 
I could add some more options like include a image in the content. But i am not sure if that is a good thing to add.

Park0 committed:
[Marktplaats] Added img & raw support

The advertisement image can now be optional added to the content
The raw JSON can now be optional added to the content

Park0 commented:
I just added the 2 parameters

Park0 commented:
@teromene @em92 anything i can do to get this merged?

teromene commented:
Sorry, I totally forgot with the rewrite of the TwitterBridge

Park0 commented:
thx @teromene
